pkg/sql/sqlbase/table.go

```diff
@@ -536,3 +536,29 @@ func ConditionalGetTableDescFromTxn(
 	}
 	return existingKV.Value, nil
 }
+
+// GetTableDescFromTxn returns the TableDescriptor and the corresponding
+// roachpb.Value written to kv, for the supplied findTable.
+func GetTableDescFromTxn(
+	ctx context.Context, txn *kv.Txn, findTable *TableDescriptor,
```

Comment:
nit: I'd pass TableID not the whole desc, since we're looking for a desc.

Reply:
done.